<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>newTask_agenda_ui</class>
 <widget class="QWidget" name="newTask_agenda_ui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>568</width>
    <height>278</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Nueva Tarea</string>
  </property>
  <property name="windowIcon">
   <iconset resource="rc.qrc">
    <normaloff>:/new/sign-up.png</normaloff>:/new/sign-up.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="1" column="0">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>15</width>
         <height>5</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="0">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="pushButton_Acept">
         <property name="text">
          <string>Aceptar</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="pushButton_Save">
         <property name="text">
          <string>Guardar</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QPushButton" name="pushButton_Cancel">
         <property name="text">
          <string>Cancelar</string>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="0" column="0">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="4" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Historia Clínica </string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="lineEdit_historiaClinica">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="4" column="3" colspan="4">
        <widget class="QLineEdit" name="lineEdit_Telefono"/>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Descripción </string>
         </property>
        </widget>
       </item>
       <item row="5" column="1" colspan="6">
        <widget class="QLineEdit" name="lineEdit_Descripcion"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Buscar por:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="6">
        <widget class="QPushButton" name="pushButton_apellido">
         <property name="text">
          <string>Buscar</string>
         </property>
         <property name="icon">
          <iconset resource="rc.qrc">
           <normaloff>:/new/zoom.png</normaloff>:/new/zoom.png</iconset>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="7">
        <widget class="QTableWidget" name="tableWidget">
         <column>
          <property name="text">
           <string>Nombre</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Historia Clínica</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Teléfono</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Celular</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Fecha:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QDateEdit" name="dateEdit">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="displayFormat">
          <string>yyyy/MM/dd</string>
         </property>
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Hora:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="4" colspan="3">
        <widget class="QTimeEdit" name="timeEdit">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="displayFormat">
          <string>hh:mm:ss</string>
         </property>
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Nombre:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="6">
        <widget class="QLineEdit" name="lineEdit_Name"/>
       </item>
       <item row="2" column="2" colspan="4">
        <widget class="QLineEdit" name="lineEdit_PacienteSearch"/>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="comboBox_search">
         <item>
          <property name="text">
           <string>Historia Clínica</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Nombre</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Dni</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Teléfono / Celular</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>dateEdit</tabstop>
  <tabstop>timeEdit</tabstop>
  <tabstop>lineEdit_Name</tabstop>
  <tabstop>comboBox_search</tabstop>
  <tabstop>lineEdit_PacienteSearch</tabstop>
  <tabstop>pushButton_apellido</tabstop>
  <tabstop>tableWidget</tabstop>
  <tabstop>lineEdit_historiaClinica</tabstop>
  <tabstop>lineEdit_Telefono</tabstop>
  <tabstop>lineEdit_Descripcion</tabstop>
  <tabstop>pushButton_Acept</tabstop>
  <tabstop>pushButton_Save</tabstop>
  <tabstop>pushButton_Cancel</tabstop>
 </tabstops>
 <resources>
  <include location="rc.qrc"/>
 </resources>
 <connections/>
</ui>
